app/(frontend)/page.tsx

```
@@ -5,30 +5,36 @@ import LastSeen from "@/components/LastSeen";
       </div>
       <div className="md:w-1/2 pt-16">
         <SectionContainer title="Now">
           <Paragraph classname="font-inter font-normal text-primary-text">
-            Currently working as a Senior Experience Engineer at{" "}
-            <a
-              href="https://www.publicissapient.com/"
-              className="duration-500 hover:text-yellow-500 underline"
-            >
-              Publicis Sapient
-            </a>
+            {now && now[0].nowCompnayDescription}{" "}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)